### PR TITLE
Remove unstable feature: mpsc_select.

### PR DESCRIPTION
**Pull request by chaoticlonghair**

### CODE REVIEW

**Review by KaoImin — APPROVED** [review]
a good method replaced the select! even if it loses a bit of efficiency

File: src/main.rs [hunk 1]
```diff
@@ -183,10 +188,10 @@ fn main() {
     );
     thread::spawn(move || loop {
         let (key, body) = rx_sub.recv().unwrap();
-        let tx = mq2main.clone();
         let pool = threadpool.clone();
+        let tx = sender.clone();
         pool.execute(move || {
-            tx.send((key, body)).unwrap();
+            tx.lock().send(BftTurn::Message((key, body))).unwrap();
         });
```
Comment (TheWaWaR) [inline src/main.rs:191]: Why `threadpool` here? `send` is a very cheap function, just call it?

Comment (chaoticlonghair) [reply src/main.rs:191]: Maybe you should ask the folk who add the `threadpool`.
I do not know why he/her do that.

Comment (u2) [reply src/main.rs:191]: Is `lock` necessary? This may introduce the deadlock problem. I reckon that `Arc::clone()` is ok here.

File: Cargo.toml [hunk 2]
```diff
@@ -28,6 +28,7 @@ authority_manage = { git = "https://github.com/cryptape/cita-common.git", branch
 rustc-hex = "1.0"
 ntp = "0.4.0"
 min-max-heap = "1.2.2"
+parking_lot = "0.6"
```
Comment (driftluo) [inline Cargo.toml]: `parking lot` has been released 0.7 for some time

Comment (chaoticlonghair) [reply Cargo.toml]: To keep the same version with other dependencies.

File: src/main.rs (hunk 1 above)
Comment (u2) [reply src/main.rs:191]: @TheWaWaR @yangby-cryptape Another PR for the `threadpool` problem.

Now it looks good for me.

**Review by u2 — APPROVED** [review]

**Pull request merged**